integration/unplugin/src/index.ts

```diff
@@ -178,7 +178,7 @@ const civetUnplugin = createUnplugin((options: PluginOptions = {}) => {
 
       const relativeId = path.relative(
         process.cwd(),
-        path.resolve(path.dirname(importer ?? ''), id)
+        path.join(path.dirname(importer ?? ''), id)
```

Comment:
Actually, I wonder whether this is correct... I would have guessed:
* If the path is relative, use `path.join` with `importer`, as you have it here
* If the path is absolute, use `path.join` with the project's `root` setting (which probably also needs to be `join`ed with `process.cwd()` because it too can be relative).

This is somewhat confirmed because I find `src/main.civet` can successfully import `/module.civet`, but I don't think it should be able to; it should have to import `/src/module.civet` I believe.

Reply:
Oops yeah you're correct, I overlooked that case.